tests/test_endpoint_buffers.py

```diff
@@ -272,8 +486,37 @@ def test_update_stats_forward_markets(self, forward_setup):
 
 
 class TestCoefficientEndpointBuffer(TestSimulationEndpointBuffer):
-    """Tests for the CoefficientEndpointBuffer class.
+    """Tests for the CoefficientEndpointBuffer class."""
+    def test_update_stats_scm(self, scm_setup):
+        CoefficientArea, _ = scm_setup
```

Comment:
CamelCase should only be used for [classes](https://peps.python.org/pep-0008/#class-names). For normal variables and fucntions we use [lowercase](https://peps.python.org/pep-0008/#function-and-variable-names)
```suggestion
        coefficient_area, _ = scm_setup
```

Reply:
got it!